public/app/features/dashboard-scene/saving/DashboardSceneChangeTracker.ts

```diff
@@ -188,5 +188,6 @@ export class DashboardSceneChangeTracker {
   public terminate() {
     this.stopTrackingChanges();
     this._changesWorker?.terminate();
+    this._changesWorker = undefined;
```

Comment:
Good catch Kay! Can we update the tests to ensure, after terminating the tracker, the worker is undefined? Just to avoid regressions :)